MyString.java

```diff
@@ -49,8 +77,13 @@ public static boolean subsetOf(String str1, String str2) {
      * @return a string consisting of the characters of str, separated by spaces.
      */
     public static String spacedString(String str) {
-        //// Replace the following statement with your code
-        return null;
+        String spacedStr = "";
+        for (int i = 0; i < str.length(); i++) {
+            spacedStr += str.charAt(i); 
+            spacedStr += (i + 1 < str.length() ? " " : "");

```

Comment:
Could merge to one line
`spacedStr += str.charAt(i) + (i + 1 < str.length() ? " " : "");`
or 
`spacedStr += (i + 1 < str.length() ? str.charAt(i) + " " : str.charAt(i));`

Reply:
thanks